contracts/Whitelist.sol

```diff
@@ -9,8 +11,23 @@ import "./packages/oz/Ownable.sol";
  * @notice The whitelist module keeps track of all valid Otoken contracts.
  */
 contract Whitelist is Ownable {
-    ///@notice mapping to track whitelisted product
+    ///@notice AddressBook module address
+    address public addressBook;
+
+    ///@dev mapping to track whitelisted product
     mapping(bytes32 => bool) internal whitelistedProduct;
+    ///@dev mapping to track whitelisted otoken
+    mapping(address => bool) internal whitelistedOtoken;
+
+    /**
+     * @dev constructor
+     * @param _addressBook AddressBook module address
+     */
+    constructor(address _addressBook) public {
+        require(_addressBook != address(0), "Invalid address book");
+
+        addressBook = _addressBook;
+    }
```

Comment:
Can anyone call the constructor or just the owner?

Reply:
constructor called only at deployment time.